src/__tests__/addTask.test.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @jest-environment jsdom
+ */
+
+// import { add } from 'lodash';
+import addTask from '../__mocks__/addTask.js';
+import localStorageMock from '../__mocks__/localStorage.js';
```

Comment:
- Fix these errors displayed in console:
![Screen Shot 2021-09-29 at 9 54 31](https://user-images.githubusercontent.com/22258274/135294162-35a6f4fe-167f-42a6-b728-a0bee2f7a004.png)

Reply:
This issue is well resolved! ✔️